src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2061,23 +2059,11 @@ namespace NServiceBus.Pipeline
         public abstract System.Threading.Tasks.Task Invoke(TFrom context, System.Func<TTo, System.Threading.Tasks.Task> next);
         public virtual System.Threading.Tasks.Task Warmup() { }
     }
-    public struct StepEnded
```

Comment:
I don't think we need it since you could only really use `StepEnded` with `StepStarted` and `StepStarted` was only exposed on the `PipelineNotifications` which is properly obsoleted. What is your take?